beta/src/pages/learn/add-react-to-a-website.md

```diff
@@ -182,93 +184,93 @@ with:
 return <button onClick={() => setLiked(true)}>Like</button>;
 ```
 
-2. In **index.html**, add `type="text/babel"` to the like button's script tag:
+2. **index.html** में, लाइक बटन के script टैग में `type="text/babel"` ऐड करें:
 
 ```html
 <script src="like_button.js" type="text/babel"></script>
 ```
 
-Here is [an example HTML file with JSX](https://raw.githubusercontent.com/reactjs/reactjs.org/main/static/html/single-file-example.html) that you can download and play with.
+यहाँ [JSX के साथ एक उदाहरण HTML फ़ाइल](https://raw.githubusercontent.com/reactjs/reactjs.org/main/static/html/single-file-example.html) है जिसे आप डाउनलोड कर सकते हैं और उसके साथ खेल सकते हैं।
 
-This approach is fine for learning and creating simple demos. However, it makes your website slow and **isn't suitable for production**. When you're ready to move forward, remove this new `<script>` tag and the `type="text/babel"` attributes you've added. Instead, in the next section you will set up a JSX preprocessor to convert all your `<script>` tags automatically.
+सरल डेमो सीखने और बनाने के लिए यह दृष्टिकोण ठीक है। हालांकि, यह आपकी वेबसाइट को धीमा कर देता है और **प्रोडक्शन के लिए उपयुक्त नहीं है**। जब आप आगे बढ़ने के लिए तैयार हों, तो यह नया `<script>` टैग और आपके द्वारा ऐडी गई `type="text/babel"` एट्रिब्यूट को हटा दें। इसके बजाय, अगले भाग में आप अपने सभी `<script>` टैग को स्वचालित रूप से बदलने के लिए एक JSX प्रीप्रोसेसर स्थापित करेंगे।
 
-### Add JSX to a project {/*add-jsx-to-a-project*/}
+### किसी प्रोजेक्ट में JSX ऐड करें {/*add-jsx-to-a-project*/}
 
-Adding JSX to a project doesn't require complicated tools like a [bundler](/learn/start-a-new-react-project#custom-toolchains) or a development server. Adding a JSX preprocessor is a lot like adding a CSS preprocessor.
+किसी प्रोजेक्ट में JSX को ऐडने के लिए [बंडलर] (/learn/start-a-new-react-project#custom-toolchains) या एक डेवलपमेंट सर्वर जैसे जटिल टूल की आवश्यकता नहीं होती है। JSX प्रीप्रोसेसर ऐडना एक CSS प्रीप्रोसेसर ऐडने जैसा है।
```

Comment:
`ऐडना` => `ऐड करना`

Reply:
@Anishpras I think you removed this part of the line `JSX प्रीप्रोसेसर ऐड करना एक CSS प्रीप्रोसेसर ऐड करने जैसा है।`

Can you add it back at line 197